examples/tf/dqn_pong.py

```diff
@@ -24,8 +24,18 @@
 from garage.tf.q_functions import DiscreteCNNQFunction
 
 
-def run_task(snapshot_config, *_):
-    """Run task."""
+def run_task(snapshot_config, variant_data, *_):
```

Comment:
`*_` _is_ variant_data

Reply:
Acc to this:
https://github.com/rlworkgroup/garage/blob/02076981cd07b2aeee44e57bb6d19b0195426a9a/src/garage/experiment/experiment_wrapper.py#L165-L166
Wouldn't `*_` be `[variant_data, args.resume_from_dir, args.resume_from_epoch]`? Should I use `_[0]['buffer_size']` instead?

Reply:
I stand corrected.

Reply:
note that `*_` should appear as that in the docstring (not just `_`)

Reply:
Not sure I get it. Are you saying `[variant_data, args.resume_from_dir, args.resume_from_epoch]` collectively represent variant_data?

I changed it from `_` to `*_` in docstring.